java/developer-guide/working-with-smtp-client/utility-features-smtp-client/_index.md

```diff
@@ -1,113 +1,55 @@
 ---
 title: Utility Features - SMTP Client
 type: docs
-weight: 40
+weight: 30
 url: /java/utility-features-smtp-client/
 ---
 
+# Utility Features - SMTP Client

```

Comment:
Remove this line. 
`title: Utility Features - SMTP Client` creates H1 automatically.